core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy

```diff
@@ -0,0 +1,51 @@
+package jenkins.model.GlobalCloudConfiguration
+
+import hudson.slaves.Cloud
+
+
+def f = namespace(lib.FormTagLib)
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName) {
+    l.side_panel {
+        l.tasks {
+            l.task(icon:"icon-up icon-md", href:rootURL+'/', title:_("Back to Dashboard"))
+            l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/computer/", title:_("Manage Nodes"))
+        }
+    }
+    l.main_panel {
+        h1 {
+            l.icon(class: 'icon-health-40to59 icon-xlg')
+            // TODO more appropriate icon
+            text(my.displayName)
+        }
+
+        def clouds = Cloud.all()
+
+        if (!clouds.isEmpty()) {
+            p()
```

Comment:
Just out of curiosity, what's the reason for putting an empty `p`  tag before the other content?

Reply:
Unsure. Copied straight from https://github.com/jenkinsci/jenkins/blob/0af98fe7163a821710f8be0dbc11086d4acd8bf2/core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy#L24